testall.sh

```diff
@@ -95,11 +102,13 @@ Check() {
 
     generatedfiles=""
 
-    generatedfiles="$generatedfiles ${basename}.ll ${basename}.s ${basename}.exe ${basename}.out" &&
-    Run "$FIRE" "$1" ">" "${basename}.ll" &&
+    generatedfiles="$generatedfiles ${basename}.ll ${basename}.s" &&
+    echo "Compiling intermediate IR LLVM code..." &&
+    Run "$FIRE" "<" "$1" ">" "${basename}.ll" &&
+    echo "IR Code generated!" &&
     Run "$LLC" "-relocation-model=pic" "${basename}.ll" ">" "${basename}.s" &&
-    Run "$CC" "-o" "${basename}.exe" "${basename}.s" "arrlib.o" "filelib.o" "regexlib.o" "printlib.o" &&
-    Run "./${basename}.exe" > "${basename}.out" &&
+    Run $CC "-o" "${basename}.flames" "${basename}.s" $PRINTL $REGX $ARRL $FIL $UTL &&
```

Comment:
Why did we drop `"` here?

Reply:
Not sure why they got dropped but they've been added again.